apps/crn-server/src/data-providers/contentful/user.data-provider.ts

```diff
@@ -433,6 +462,14 @@ export const parseContentfulGraphQlUserListItem = (
     onboarded: typeof item.onboarded === 'boolean' ? item.onboarded : true,
     role: item.role && isUserRole(item.role) ? item.role : 'Guest',
     teams: userTeams,
+    tags:
+      item.tagsCollection?.items
```

Comment:
you might want to extract this and reuse across two parsers for detail and list view